Framework/MDAlgorithms/src/IntegratePeaksMD2.cpp

```diff
@@ -118,7 +120,22 @@ void IntegratePeaksMD2::init() {
                   "BackgroundInnerRadius + AdaptiveQMultiplier * **|Q|**");
 
   declareProperty("Ellipsoid", false, "Default is sphere.");
-  declareProperty("FixQAxis", false, "Default is sphere.");
+  declareProperty("FixQAxis", false,
+                  "Fix one axis of ellipsoid to be along direction of Q.");
+  declareProperty("FixMajorAxisLength", true,
+                  "Set major axis to be equal to PeakRadius and scale the "
+                  "length of the orthogonal axes by proportional to the sqrt "
+                  "of the eigenvalues of the covariance matrix (this ignored "
+                  "if all three peak radii are specified)");
+  declareProperty("UseCentroid", false,
+                  "Perform integration on estimated centroid not peak position "
+                  "(ignored if all three peak radii are specified).");
+  auto maxIterValidator = std::make_shared<BoundedValidator<int>>();
+  maxIterValidator->setLower(1);
+  declareProperty(
+      "MaxIterations", 1, maxIterValidator,
+      "Number of iterations in covariance estimation (ignored if all "
+      "peak radii are specified). 2-3 should be sufficient.");
```

Comment:
Sorry I forgot about that - I'm looking into putting the ellipsoid arguments in a group so they are displayed together in the dialog box even if these options are put at the end. Otherwise I think it will be confusing and I suspect (though I don't know!) no one is using positional args for this function as it has so many inputs for different modes of operation - so perhaps if the grouping doesn't work we can discuss .

Reply:
The dialog box looks like this - even though the properties are part of the same group the new properties still appear at the bottom. I think this is fine for now (at least it's clear which properties relate to ellipsoids) but perhaps we can sort this in issue #30797?
![image](https://user-images.githubusercontent.com/55979119/110121194-010bc300-7db6-11eb-9dec-9ddd8585d74c.png)

Reply:
I'm not sure we will be able to do anything about the ordering in #30797. I think it would require a core change to improve on this rather than there being something we can achieve by tinkering with the existing settings that are available. So we may as well decide on what to do now. 

One option is that we check with Martyn and anyone who might use this algorithm (SNS\WISH) if they're happy for you to reorder the parameters. If you're happy to do this bit of asking around and they're happy then I'm also fine. With that approach we'd just advertise in the release notes that we're making a "breaking" change that could impact existing python scripts.

Alternatively we could just live with the fact the parameters are in a funny order on the dialog.

Reply:
I'm happy to live with the funny order, what do you think @martyngigg?

Reply:
One question regarding the parameter reordering. If we went with that would it be likely that a script would just break or silently do something else? The second I'm quite a lot more nervous about.

In any case I'd be happy to create a core issue to improve the display of grouped properties as that is worth doing I think.

Reply:
I think as per @DannyHindson comment the current ordering would not break any scripts (the new parameters have now been put to the end so positional arguments in previous scripts will still be valid). It just looks messy in the dialog box - but at least it's clear which properties relate to ellipsoids.

Reply:
@martyngigg I have made an issue (#30836) for displaying grouped properties together - hope that's OK